Gemfile.lock

```diff
@@ -403,14 +404,14 @@ GEM
       crass (~> 1.0.2)
       nokogiri (>= 1.4.4)
       nokogumbo (~> 1.4.1)
-    sidekiq (4.2.10)
+    sidekiq (5.0.0)
```

Comment:
I've read through the changelog/upgrade notes, we're probably fine. I'm not sure if delayed mailers would be affected by the deprecation/removal of the "delay" stuff